src/app.css

```
@@ -6,8 +6,8 @@
 body {
   background-image: linear-gradient(
     180deg,
-    rgba(179, 54, 54, 1) 0%,
-    rgba(65, 19, 21, 1) 25rem,
+    rgba(179, 54, 54, 1) -10%,
```

```suggestion
    rgba(155, 47, 48, 1) 0%,
```
To avoid the negative percentage in order for it to be more readable, we can just start the gradient at a darker color. I just eyedropped the top color when the gradient had a negative position and inserted it :)

---

Done and thank you :)